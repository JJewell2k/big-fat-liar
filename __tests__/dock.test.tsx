import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import App from '../pages/_components/Dock/index'

describe('Dock component', () => {
  let app: ShallowWrapper

  beforeEach(() => {
    app = shallow(<App />)
  })

  it('Should renders properly', () => {
    expect(shallowToJson(app)).toMatchSnapshot()
  })
})
