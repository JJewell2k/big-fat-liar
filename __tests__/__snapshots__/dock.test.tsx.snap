// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dock component Should renders properly 1`] = `
<div
  className="flex h-16 flex-row justify-center items-end bg-white fixed bottom-2 left-0 right-0 px-2 bg-opacity-10 w-max m-auto rounded-xl"
>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Finder"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/finder-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Safari"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/safari-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Messages"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/messages-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Music"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/music-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Mail"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/mail-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Photos"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/photos-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Contacts"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/contacts-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Calendar"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/calendar-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Stocks"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/stocks-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Facetime"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/facetime-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Maps"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/maps-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Note"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/note-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Settings"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/settings-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="Reminders"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/reminders-logo.png"
    />
  </button>
  <button
    className="w-16 align-bottom dock-item p-2"
    key="News"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "transition": "all ease .2s",
      }
    }
  >
    <img
      alt="dock icon"
      className="select-none w-full"
      src="images/news-logo.png"
    />
  </button>
</div>
`;
